sdk/appcenter/src/test/java/com/microsoft/appcenter/IsRunningInAppCenterTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.appcenter;
```

Comment:
Needs to moved to `com.microsoft.appcenter.utils`

Reply:
That's more fiddly that I expected it to be - suggest we keep the first 3 Tests which specifically call `AppCenter` API in this package and move the other 2 Tests (which are really concerning InstrumentationRegistryHelper directly in to the `InstrumentationRegistryHelperTest.java` file so they more naturally sit within that package.

Reply:
sounds good